content/en/docs/about/community.md

```diff
@@ -5,30 +5,23 @@ weight = 10
 aliases = ["/docs/community/"]
 +++
 
-## Kubeflow Slack
-
-The Official Kubeflow Slack is used for informal discussions among users and contributors.
-
-<a href="https://communityinviter.com/apps/kubeflow/slack">
-  <button class="btn btn-primary py-2 px-5 mb-3">Click to join:<br>Kubeflow Slack</button>
-</a>
-
-| Popular Kubeflow Slack Channels |
-| --- | --- |
-| General Discussion | [#general](https://kubeflow.slack.com/archives/C7REE0ETX)
-| Feature Requests | [#feature-requests](https://kubeflow.slack.com/archives/C01A7RYEYMB)
-| Job Postings | [#job-postings](https://kubeflow.slack.com/archives/CJ9PJE5FS)
-| Kubeflow - Pipelines | [#kubeflow-pipelines](https://kubeflow.slack.com/archives/CE10KS9M4)
-| Kubeflow - Notebooks | [#kubeflow-notebooks](https://kubeflow.slack.com/archives/CESP7FCQ7)
-| Kubeflow - Training Operator | [#kubeflow-training-operator](https://kubeflow.slack.com/archives/C985VJN9F)
-| Kubeflow - Katib | [#kubeflow-katib](https://kubeflow.slack.com/archives/CE0BURK1B)
-| KServe | [#kserve](https://kubeflow.slack.com/archives/CH6E58LNP)
-| Platform - AWS | [#platform-aws](https://kubeflow.slack.com/archives/CKBA5D0MU)
-| Platform - Azure | [#platform-azure](https://kubeflow.slack.com/archives/CUW6SLCPR)
-| Platform - GCP | [#platform-gcp](https://kubeflow.slack.com/archives/CKH7V1M7F)
-| Users - China | [#users-china](https://kubeflow.slack.com/archives/C93HYNM9C)
-| Users - Korea | [#users-korea](https://kubeflow.slack.com/archives/CKPCJB9AP)
-| Users - Oceania | [#users-oceania](https://kubeflow.slack.com/archives/C023ZN1R9FC)
+## Kubeflow Slack Channels
+
+Kubeflow is part of the vibrant CNCF community. You are welcome to create your CNCF Slack account [here](https://slack.cncf.io/) and then join the following list of Kubeflow Slack channels.
+
+| Description                 | Link                                                                             |
+|-----------------------------|----------------------------------------------------------------------------------|
+| Announcements               | [#kubeflow-announcements](https://app.slack.com/client/T08PSQ7BQ/C01EV0FV154)    |
+| GSoC Mentors                | [#kubeflow-gsoc-mentors](https://app.slack.com/client/T08PSQ7BQ/C06LJP14PB8)     |
+| GSoC Participants           | [kubeflow-gsoc-participants](https://app.slack.com/client/T08PSQ7BQ/C0742LBR5BM) |
+| Katib                       | [#kubeflow-katib](https://app.slack.com/client/T08PSQ7BQ/C073N7AS48P)            |
+| Model Registry              | [#kubeflow-model-registry](https://app.slack.com/client/T08PSQ7BQ/C073N7B6K3R)   |
+| Notebooks                   | [#kubeflow-notebooks](https://app.slack.com/client/T08PSQ7BQ/C073W562HFY)        |
+| Pipelines                   | [#kubeflow-pipelines](hhttps://app.slack.com/client/T08PSQ7BQ/C073N7BMLB1)       |
+| Release Team Communications | [#kubeflow-release](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2)          |
```

Comment:
Should we rename kubeflow-release channel to kubeflow-platform ? I don't think we need to have 2 channels and Kubeflow release is always associated  with Kubeflow Platform release. 
Thoughts @kubeflow/wg-manifests-leads @kubeflow/release-team ?

Reply:
Oh I think we might have discussed that in the original spreadsheet already and decided to use kubeflow-platform instead. Need to double check.

Reply:
Just checked the comments on the spreadsheet. 

#kubeflow-release is a separate channel. 
#wg-manifests is suggested to be #kubeflow-platform and we are postponing that. 

Let me know if we need to change the plan here.

Reply:
@terrytangyuan Can we just start with a single #kubeflow-platform channel for release planning + manifests discussion. We can create separate channel for release in the future if that will be required.

Reply:
I think `#kubeflow-release` is important to have, its a very active channel on the old slack which we use to discuss things related to the release.

Calling it anything else will make it harder for people to migrate.

Reply:
We can make this new `#kubeflow-platform` channel very active and post release updates there.
As I said, these releases are associated with Kubeflow manifests release, so they are related to the Kubeflow Platform discussion.
Release updates for standalone Kubeflow components should be discussed in the components channels.

What do you think @kubeflow/release-team @kubeflow/wg-manifests-leads @diegolovison @hbelmiro @nagar-ajay @stevei101 @milosjava @ca-scribner ?

Reply:
@andreyvelich  sounds good!

Reply:
I'm fine with #kubeflow-platform for both release and manifests discussion.

Reply:
Renamed the channel to be kubeflow-platform